apps/meteor/app/mentions/server/methods/getUserMentionsByChannel.js

```diff
@@ -24,6 +25,6 @@ Meteor.methods({
 			});
 		}
 
-		return Messages.findVisibleByMentionAndRoomId(user.username, roomId, options).fetch();
+		return Messages.findVisibleByMentionAndRoomId(user.username, roomId, options).toArray();
```

Comment:
In raw model, this uses findpaginated

Reply:
well, that explains the failed test 😬

Reply:
should be fixed now.. created both paginated/non-paginated variantes.